data_collection/gazette/spiders/pr_guarapuava.py

```diff
@@ -0,0 +1,58 @@
+import re
+from datetime import date, datetime
+
+import scrapy
+from dateparser import parse
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrGuarapuavaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4109401"
+    name = "pr_guarapuava"
+    allowed_domains = ["guarapuava.pr.gov.br"]
+    start_urls = ["https://www.guarapuava.pr.gov.br/boletins-oficiais"]
+    start_date = datetime(2002, 1, 20).date()
```

Comment:
**suggestion**: You can create a `date` directly instead of converting a `datetime`to date:
`start_date = date(2002, 1, 20)`